frontend/src/employee-frontend/components/messages/MessagesPage.tsx

```diff
@@ -19,7 +19,19 @@ import { deselectAll, SelectorNode } from './SelectorNode'
 import Sidebar from './Sidebar'
 import { MessageBody } from './types'
 
+// TODO is fixed header height possible?
+// If not, replace with a stretching flex container with scrollable children
+const approximatedHeaderHeight = `164px`
+const MessagesPageContainer = styled(Container)`
+  height: calc(100vh - ${approximatedHeaderHeight});
+`
```

Comment:
I played around with this for an hour so but couldn't come up with a solution that would retain the correct page width while always occupying all the available height.